DataConnectors/CrowdstrikeFDR/azuredeploy_Connector_CrowdstrikeFalconAPI_AzureFunction.json

```diff
@@ -0,0 +1,206 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "FunctionName": {
+            "defaultValue": "CSFalcon",
+            "minLength": 1,
+            "maxLength": 11,
+            "type": "string"
+        },
+        "WorkspaceID": {
+            "type": "string",
+            "defaultValue": "<workspaceID>"
+        },
+        "WorkspaceKey": {
+            "type": "securestring",
+            "defaultValue": "<workspaceKey>"
+        },
+        "AWS_KEY": {
+            "type": "string",
+            "defaultValue": "<AWS_KEY>"
+        },
+        "AWS_SECRET": {
+            "type": "securestring",
+            "defaultValue": "<AWS_SECRET>"
+        },
+        "AWS_REGION_NAME": {
+            "type": "string",
+            "defaultValue": "<AWS_REGION_NAME>"
+        },
+        "QUEUE_URL": {
+            "type": "string",
+            "defaultValue": "<QUEUE_URL>"
+        }
+    },
+    "variables": {
+        "FunctionName": "[concat(toLower(parameters('FunctionName')), uniqueString(resourceGroup().id))]",
+         "StorageSuffix": "[environment().suffixes.storage]",
+         "LogAnaltyicsUri": "[replace(environment().portal, 'https://portal', concat('https://', toLower(parameters('WorkspaceID')), '.ods.opinsights'))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Insights/components",
+            "apiVersion": "2015-05-01",
+            "name": "[variables('FunctionName')]",
```

Comment:
Can you please confirm that it is parameters instead of variables?

Reply:
@v-maudan What do you mean? variables('FunctionName') - it is variable based on Function name parameter + uniq string(res group ID)